.eslintrc.json

```diff
@@ -20,6 +20,12 @@
         "no-fallthrough": "error",
         "no-new-wrappers": "error",
         "no-plusplus": "error",
+        "valid-typeof": "error",
```

Comment:
Looks like you did not merge the develop branch in the correct way. Now it contains many unrelated changes from other PR.

Reply:
I did pull the latest code from dev